tendrl/commons/flows/import_cluster/__init__.py

```diff
@@ -21,15 +21,28 @@ def run(self):
             
             # If cluster.import_status="failed", allow retries
             try:
-                NS._int.wclient.delete(_cluster_import_status, prevValue="failed")
-                NS._int.wclient.delete(_cluster_import_job_id)
+                NS._int.wclient.write(
+                    _cluster_import_status,
+                    "",
+                    prevValue="failed",
+                    prevExist=True)
+                NS._int.wclient.write(
+                    _cluster_import_job_id,
+                    "",
+                    prevExist=True)
             except (etcd.EtcdKeyNotFound, etcd.EtcdCompareFailed):
                 pass
             
             try:
 
-                NS._int.wclient.write(_cluster_import_status, "in_progress", prevExist=False)
-                NS._int.wclient.write(_cluster_import_job_id, self.job_id, prevExist=False)
+                NS._int.wclient.write(
+                    _cluster_import_status,
+                    "in_progress",
+                    prevExist=True)
```

Comment:
prevExist=True means  this mark import_status as "in_progress" even if it is "done", "in_progress" or "failed"

Why would you even want this? This code is about locking a cluster import, your change doesnt do anything related to that